akka-docs/src/main/paradox/additional/native-image.md

```diff
@@ -0,0 +1,86 @@
+---
+project.description: How to build native image executables with Akka
+---
+# Building Native Images
+
+Building native images with Akka is supported, both for local and cluster actor system applications. 
+
+Most built-in features in Akka can be used as is, however some functionality is not available and some extension points
+require additional metadata for reflective access to work.
+
+## Unsupported features
+
+The following features cannot be used in native image apps:
+
+* Lightbend Telemetry
+* Aeron UDP remoting
+* Testkits
+* LevelDB and InMem Akka Persistence plugins
+* @ref[Durable storage](../typed/distributed-data.md#durable-storage) for Akka Distributed Data
+
+## Features requiring additional metadata
+
+In general, any feature that allows a custom implementation provided by a third party library or user code that is
+then plugged into Akka through an entry in the application config file needs to have reflection metadata explicitly added
+as described in the [GraalVM docs here](https://www.graalvm.org/latest/reference-manual/native-image/metadata/).
+
+Examples of such classes are:
+
+ * Custom Serializers
+ * Custom Akka Extensions
```

Comment:
That is only if they are loaded from config, right? Clarify?

Reply:
Yeah, when I think of it, the sample didn't need any extra reflection config, so the whole thing about `Feature` for extensions was wrong. If you load the extension by interacting with it, no reflection is needed. I'll amend this.

Reply:
Actually, let's drop if from this bullet list, there is a whole paragraph about it below.